teammapper-frontend/mmp/src/map/handlers/draw.ts

```diff
@@ -288,6 +298,35 @@ export default class Draw {
         }
     }
 
+    /**
+     * Set a hidden eye icon if child nodes are hidden.
+     * @param {Node} node
+     */
+    public setHiddenChildrenIcon(node: Node) {
```

Comment:
maybe we can extract parts of the method as its similar to the link icon?

Reply:
Don't think so personally, there's just a little bit of overlap and it's two very different things at the end of the day (setting a link when clicked vs automatically setting a very specific icon when conditions apply)

Reply:
i am only referring to the icon, not to the logic. e.g. creating an element, setting the icon name, setting the position according to parameters etc. Thats not just one line. BTW there is I think a copy paste error: `domText.classList.add('link-text')` ;-)

Reply:
Yeah `link-text` was unnecessary, removed.